.eslintrc

```diff
@@ -13,7 +13,29 @@
     "react"
   ],
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "arrowFunctions": true,
+    "binaryLiterals": true,
+    "blockBindings": true,
+    "classes": true,
+    "defaultParams": true,
+    "destructuring": true,
+    "forOf": true,
+    "generators": true,
```

Comment:
Agree with @hxpaul: if it violates best practices and can be caught by the linter, then it should be enforced by the linter. We should automate anything that can be automated.